lib/src/core/channel/guild/guild_channel.dart

```diff
@@ -157,10 +171,24 @@ abstract class GuildChannel extends MinimalGuildChannel implements IGuildChannel
   Future<void> deleteChannelPermission(SnowflakeEntity entity, {String? auditReason}) =>
       client.httpEndpoints.deleteChannelPermission(id, entity, auditReason: auditReason);
 
-  /// Creates new [Invite] for [IChannel] and returns it"s instance
+  /// Creates new [Invite] for [IChannel] and returns it's instance.
+  ///
+  /// Requires the `CREATE_INSTANT_INVITE` permission.
+  ///
+  /// [maxAge] in seconds can be between 0 and 604800 (7 days).
+  /// If set to 0, the invite will never expire. The default is 24 hours.
+  ///
+  /// [maxUses] can be between 0 and 100. If set to 0, the invite will have
+  /// unlimited uses. The default is 0 (unlimited).
+  ///
+  /// [temporary] determines whether this invite only grants temporary
+  /// membership.
+  ///
+  /// If [unique] is true, Discord won't try to reuse a similar invite and will
+  /// instead always generate a new invite in the request.
   ///
   /// ```
-  /// var invite = await channel.createInvite(maxUses: 2137);
+  /// var invite = await channel.createInvite(maxUses: 100);
   /// ```
```

Comment:
docs needs to be only on interface. Doesn't make sense to update them on class that is not shown to end user. Can you remove all docs except this one that are on interfaces?